src/System.IO.IsolatedStorage/src/System/IO/IsolatedStorage/Helper.Win32.Unix.cs

```diff
@@ -68,6 +68,11 @@ internal static void GetDefaultIdentityAndHash(out object identity, out string h
             // by pulling directly from EntryAssembly.
 
             Assembly assembly = Assembly.GetEntryAssembly();
+
+            // Fall back on the executing assembly if needed
```

Comment:
Any ideas on what to do here instead? This happened when I tried to run against desktop.